packages/experiments/package.json

```diff
@@ -35,6 +36,7 @@
     "@uifabric/example-app-base": ">=6.2.1 <7.0.0",
     "@uifabric/file-type-icons": ">=6.0.0 <7.0.0",
     "@uifabric/jest-serializer-merge-styles": ">=6.0.3 <7.0.0",
+    "deep-assign": "^2.0.0",
```

Comment:
Actually types is ok as a devDepenency. This is the dependency that shouldn't be a dev dependency

Reply:
Cool. Makes sense. I'll just move this one over to dependencies then